sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -214,6 +214,22 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
       std::vector<Models::FileSystem> Filesystems;
     };
 
+    class PublicAccessType {
+    public:
+      PublicAccessType() = default;
+      explicit PublicAccessType(std::string value) : m_value(std::move(value)) {}
+      bool operator==(const PublicAccessType& other) const { return m_value == other.m_value; }
+      bool operator!=(const PublicAccessType& other) const { return !(*this == other); }
+      const std::string& Get() const { return m_value; }
+
+      const static PublicAccessType FileSystem;
+      const static PublicAccessType Path;
+      const static PublicAccessType Private;
```

Comment:
Should this be `Private` or `None`?

Reply:
Private to be consistent with blob.

Reply:
Blob might be wrong. I'm considering change it to None in blob. What do they do in .Net?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Files.DataLake/src/Models/PublicAccessType.cs#L14
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs#L2092
In .NET, it is None. I'm changing to None.